examples/hosts/app-integration/external-data/src/model-interface/TaskData.ts

```diff
@@ -0,0 +1,44 @@
+/*!
```

Comment:
`TaskData` was extracted from `index.ts`. Since I was adding a related utility function, it seemed worth doing.